auto_nag/tests/test_utils.py

```diff
@@ -37,3 +41,54 @@ def test_hide_personal_info(self):
             "include_fields=_default,attachments&changed_field_to=" + \
             "RESOLVED&api_key=xyzxyzxyz&resolution=FIXED"
         assert "xyzxyzxyz" not in hide_personal_info(sample_exception)
+
+    def test_urljoin(self):
+        api_root = 'https://bugzilla.mozilla.org/bzapi/'
```

Comment:
@sylvestre Most places we are repeating the urls. So, I am planning to create one more file named settings(inside bugzilla directory), so that we can add all urls, or any other configuration parameters into it.